pom.xml

```diff
@@ -89,7 +89,7 @@
         <dockerCafAuditOrg>${dockerImagePrefix}${dockerHubOrganization}${dockerOrgSeperator}</dockerCafAuditOrg>
         <dockerProjectVersion>${dockerVersionSeperator}${project.version}</dockerProjectVersion>
         <enforceBannedDependencies>true</enforceBannedDependencies>
-        <enforceCorrectDependencies>true</enforceCorrectDependencies>
+        <enforceCorrectDependencies>false</enforceCorrectDependencies>
```

Comment:
Issue with javax.annotation-api being reported as unused in caf-audit-service module, have turned this off temporarily